tm2/pkg/autofile/group.go

```diff
@@ -136,11 +136,12 @@ func (g *Group) Wait() {
 
 // Close closes the head file. The group must be stopped by this moment.
 func (g *Group) Close() {
-	g.FlushAndSync()
+	_ = g.FlushAndSync()
 
 	g.mtx.Lock()
+	defer g.mtx.Unlock()
+
 	_ = g.Head.Close()
-	g.mtx.Unlock()
 }
```

Comment:
might we at least log errors here or is not necessary?

Reply:
Honestly better to have a log rather than skip it - I've added logs for other calls as well:

[dee7806](https://github.com/gnolang/gno/pull/546/commits/dee780653e66bb54364e55559313f89764d1b45b)